mos_tests/functions/common.py

```diff
@@ -418,7 +418,8 @@ def is_volume_exists(cinder_client, uid):
     return uid in [s.id for s in cinder_client.volumes.list()]
 
 
-def create_volume(cinder_client, image_id, size=1, timeout=5):
+def create_volume(cinder_client, image_id, name='Test_volume', type=None,
```

Comment:
please don't change the sequence of parameters when you want to add new parameters to the function.